src/test/java/FakeDataService.java

```diff
@@ -3,32 +3,21 @@
 import com.easypost.easyvcr.clients.httpurlconnection.RecordableHttpsURLConnection;
 import com.easypost.easyvcr.internalutilities.json.Serialization;
 
-import java.util.List;
-
 import static com.easypost.easyvcr.internalutilities.Tools.readFromInputStream;
 
 public class FakeDataService {
 
-    public class ExchangeRates {
-        public String code;
-        public String currency;
-        public List<Rate> rates;
-        public String table;
-    }
-
-    public class Rate {
-        public String effectiveDate;
-        public float mid;
-        public String no;
+    public class IPAddressData {
+        public String IPAddress;
     }
 
-    public static final String URL = "https://api.nbp.pl/api/exchangerates/rates/a/gbp/last/10/?format=json";
+    public static final String URL = "https://api.ipify.org/?format=json";
```

Comment:
Question: Is the `?format=json` still needed here?

Reply:
Not sure, but just to make sure it is in fact returning us JSON